platform-ui-skin/src/main/webapp/skin/less/helpers.less

```diff
@@ -429,7 +429,7 @@
     border-right: 5px solid @primaryColor ~'!important; /** orientation=rt */ ';
     padding: 10px 0 10px 10px ~'!important; /** orientation=lt */ ';
     padding: 10px 10px 10px 0 ~'!important; /** orientation=rt */ ';
-    p, div {
+    p, div, ul li, ol li {
```

Comment:
I 'm not sure that the ol & li should use italic font style.
Is this the same style used in CKEditor ?
BTW, you can add more CSS files in iframe/div of CKEditor.